Source/Classes/NantesLabel.swift

```diff
@@ -542,6 +535,17 @@ public extension NSAttributedString.Key {
 
     // MARK: - Public
 
+    public func setAttributedText(_ attributedString: NSAttributedString, afterInheritingLabelAttributesAndConfiguringWithBlock block: ((NSMutableAttributedString) -> NSMutableAttributedString)?) {
```

Comment:
I would strongly consider adding documentation (for quick look and autocomplete) to this method to explain how it differs from other ways of setting attributed text.

Reply:
Updated. Good callout!

Reply:
@chansen22, another thought though I know you've merged already. You may want to have a default value of `nil` for this block to make the call site quite a bit cleaner in the case the consumer wants to pass `nil` here...

Reply:
Yeah I considered this while writing it. I think `setAttributedText(_ NSAttributedString)` is too similar to the property `attributedText` from `UILabel`, and I wanted to differentiate this function from that since we're doing a different set of work here. Similar, but different with regards to attribute setting. Hopefully by having the longer function name, we can reduce confusion around these two.